chapter02/TenArmedTestbed.py

```diff
@@ -71,7 +71,7 @@ def getAction(self):
             expEst = np.exp(self.qEst)
             self.actionProb = expEst / np.sum(expEst)
             return np.random.choice(self.indices, p=self.actionProb)
-        return argmax(self.qEst)
+        return np.argmax(self.qEst)
```

Comment:
this argmax is located in utils.py, it's not meant to be np.argmax, they are different